### ISSUE TITLE
Support aarch64 (or understand how to build a native image properly)

### ISSUE THREAD

**Issue by Aljendro** [clojure-lsp/clojure-lsp#718]
**Is your feature request related to a problem? Please describe.**
Trying to get clojure-lsp working on Linux arm (aarch64)

**Describe the solution you'd like**
Ultimately, I would like a build for Linux arm (aarch64) as a downloable file

**Describe alternatives you've considered**
I have walked through the steps to download graalvm (which is available for aarch64) and tried to
produce a native binary without success. I am trying to figure out if its a graalvm issue or clojure-lsp config issue.
Just wanted to figure out if I can succeed in compiling or if clojure-lsp just does not work on aarch64. Completely new to java/clojure ecosystems, so I do not have any intuition on this issue.

**Additional context**
![Screenshot from 2022-01-24 17-34-03](https://user-images.githubusercontent.com/7899307/150894251-3ab78de2-4b13-406e-a26d-af75ab569d42.png)

**Comment by ericdallo** [comment]
@Aljendro this is not possible ATM because datalevin, the db impl clojure-lsp uses doesn't support aarch64, but this should be no issue soon with #703 which should be available on next release

**Comment by Aljendro** [comment]
Thank you @ericdallo. This saved me going down a rabbit hole.

**Comment by ericdallo** [comment]
Yeah, I went to that hole some months ago 😂 that's one of the reason why I'd like to fix #703

**Comment by ericdallo** [comment]
@Aljendro I'm not sure how to test aarch64 easily, could you try the binary from [this run](https://github.com/clojure-lsp/clojure-lsp/actions/runs/1768851796)?
it should be on the summary section

**Comment by ericdallo** [comment]
@Aljendro, This was made on master and it should be available on next release, LMK if you find any issues
